README.md

```diff
@@ -1,59 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# CoronaWide
```

Comment:
- [ ] **[OPTIONAL]** You seem to have raised a Pull Request from `dev` to `master`. But according to the [link for correct Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) provided in your project requirements, you should raise a Pull Request from the `feature` branch to the `develop` instead. Kindly fix this.